src/EditorFeatures/CSharpTest/GenerateFromMembers/GenerateEqualsAndGetHashCodeFromMembers/GenerateEqualsAndGetHashCodeFromMembersTests.cs

```diff
@@ -20,13 +20,15 @@ namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.GenerateEqualsAndGetHas
 
     public class GenerateEqualsAndGetHashCodeFromMembersTests : AbstractCSharpCodeActionTest
     {
+        private static readonly CSharpParseOptions CSharp6 = TestOptions.Regular.WithLanguageVersion(LanguageVersion.CSharp6);
+
         protected override CodeRefactoringProvider CreateCodeRefactoringProvider(Workspace workspace, TestParameters parameters)
             => new CSharpGenerateEqualsAndGetHashCodeFromMembersCodeRefactoringProvider((IPickMembersService)parameters.fixProviderData);
 
         [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsGenerateEqualsAndGetHashCode)]
         public async Task TestEqualsSingleField()
         {
-            await TestInRegularAndScriptAsync(
+            await TestAsync(
```

Comment:
can you keep AndScriptAsync? I think `parseOptions` should respect your language version and only change script/regular. I've done something like that recently.

Reply:
Thanks!  I forgot about that.  #resolved